modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/test/java/com/liferay/dynamic/data/mapping/BaseDDMTestCase.java

```diff
@@ -428,6 +457,30 @@ protected DDMFormField createTextDDMFormField(
 			_defaultDDMFormFieldType
 		);
 
+		when(
+			ddmFormFieldTypeServicesTracker.getDDMFormFieldType("fieldset")
```

Comment:
I'm curious how these mocks behave when there's a `Matchers.anyString()` that could always return the default field type. Did you check that?
Also, could you please explain why did you choose to not use `Matchers.eq("fieldset")` instead?

Reply:
Yes, I did check, but the `DDMStructureUpgradeProcessTest` fails because of the expected JSON (`updated-ddm-structure-version-definition.json`), since each field type has its specific properties. If I remove the `fieldset`, for example, the following error happens:

<img width="1291" alt="Captura de Tela 2022-02-04 às 13 41 55" src="https://user-images.githubusercontent.com/34186632/152567768-e7d83568-2edd-402c-9dba-b91b8ac6e8ab.png">

Regarding the second question, It was actually a copy and paste of this [WIP](https://github.com/natocesarrego/liferay-portal/commit/e7a879d6b31fae4f8a380115172b98ab006ac005#diff-d23d19463504aaf36c15b20080b9d4ca3992e8b07bc608d64396dabc2fefb519R298-R313) made by Renato at that time. But I can change it, no problem 😉

Reply:
I see. I was just curious if doing `Matchers.anyString()` in one call would cause that the others wouldn't be picked up. So in other words, looks like the `Matchers.anyString()` really means "anyStringElse". 😛 
Sure thing. Thanks, @carolmariaabb!